x-pack/plugin/sql/qa/src/main/java/org/elasticsearch/xpack/sql/qa/jdbc/SqlSpecTestCase.java

```diff
@@ -13,15 +13,19 @@
 import java.net.URL;
 import java.sql.Connection;
 import java.sql.ResultSet;
+import java.time.LocalDateTime;
+import java.time.ZoneId;
 import java.util.Arrays;
 import java.util.List;
 import java.util.Locale;
+import java.util.TimeZone;
 
 /**
  * Tests comparing sql queries executed against our jdbc client
  * with those executed against H2's jdbc client.
  */
 public abstract class SqlSpecTestCase extends SpecBaseIntegrationTestCase {
+    private final int TZSYNC_MINUTE_THREASHOLD = 55;
```

Comment:
The name of the constant, I believe, has a typo in it: THREASHOLD vs THRESHOLD.

Reply:
thanks, fixed it.